Firestore/Example/Tests/Integration/API/FIRDatabaseTests.mm

```diff
@@ -969,7 +969,7 @@ - (void)testQuerySnapshotEvents_forDelete {
           XCTAssertEqual(docSet.count, 0);
           [changeCompletion fulfill];
 
-        } else if (callbacks == 4) {
+        } else if (callbacks == 3) {
```

Comment:
Good catch. 

Would `>= 3` rather than `==` make sense?

Reply:
actually let me remove the condition from the last else to catch all abnormal cases